tensorflow/core/ops/ops.pbtxt

```diff
@@ -19123,6 +19123,54 @@ op {
     }
   }
 }
+op {
```

Comment:
IIUC, what @alextp commented is that this file should not be modified.

Reply:
Solved.